packages/create/src/index.ts

```diff
@@ -144,7 +168,7 @@ export async function createFullNode(
     lightPush,
     filter,
     relay
-  ) as FullNode;
+  ) as FullNode<FilterV2 extends true ? true : false>;
```

Comment:
I'd rather avoid doing this typing and consider other approaches. For example since API is different we can have `filterv2` property on the node or have a property on `node.filter` that signifies which version it is.

Reply:
I totally understand this. I am just thinking of a way to remove this `FilterV2 extends true ? true : false` in our codebase so that consumers won't need to do `createLightNode<FilterV1>(...)`.

Since it seems that only `ping` is a [new method](https://github.com/waku-org/js-waku/pull/1302#discussion_r1178807121) we can use it to improve definitions. 

One approach can be to make `ping` optional so that if it is present then it's `FilterV2` otherwise it's `FilterV1`. Of course it can force consumers into doing `node.filter?.ping` but on the other hand everything else will be clean and filters can be used interchangeably.